api/app/models/fetch/predictions.py

```diff
@@ -13,7 +13,7 @@
                          WeatherModelPredictionValues, WeatherModelRun)
 from app.db.models import ModelRunGridSubsetPrediction
 import app.db.crud
-import app.stations
+from app.wildfire_one import get_stations_by_codes
```

Comment:
Please revert this change! Let app.stations decide if it's going to go against wf1 or local - I think this may have been a merge conflict resolved incorrectly?

```suggestion
import app.stations
```